simd/src/Kokkos_SIMD_Scalar.hpp

```diff
@@ -437,25 +437,36 @@ KOKKOS_FORCEINLINE_FUNCTION constexpr basic_simd<T, simd_abi::scalar> condition(
       static_cast<bool>(a) ? static_cast<T>(b) : static_cast<T>(c));
 }
 
-template <class T, class BinaryOperation>
+template <class T, class BinaryOperation = std::plus<>>
+KOKKOS_FORCEINLINE_FUNCTION constexpr T reduce(
+    Experimental::basic_simd<T, Experimental::simd_abi::scalar> const& x,
+    BinaryOperation binary_op = {}) noexcept {
```

Comment:
To get rid of the unused parameter warning
```suggestion
    BinaryOperation = {}) noexcept {
```

Reply:
Applied.